coding-intro/core/collections/collections-discussion-insight.md

```diff
@@ -0,0 +1,78 @@
+---
+author: kapnobatai136
+
+tags:
+  - discussion
+
+type: normal
+
+category: discussion
+
+---
+
+# Test Yourself
+
+---
+
+## Content
+
+Now that you started to learn how to code, it's time to apply yourself!
+
+How would you create a collection and store some data inside it?
+
+It's up to you to choose which type of collection you will create, and what data to store inside it.
+
+Finally, access one of the items in your collection.
+
+As a quick reminder, lists and dictionaries are considered collections.
```

Comment:
Make this sentence emphasized somehow? Two things that come to mind are

```suggestion
> 💡 As a quick reminder, lists and dictionaries are considered collections.
```

or 

```suggestion
_As a quick reminder, lists and dictionaries are considered collections._
```